scripts/deploy/08_steth_call_theta_vault.ts

```diff
@@ -44,22 +46,26 @@ const main = async ({
   );
 
   const initArgs = [
-    owner,
-    keeper,
-    feeRecipient,
-    MANAGEMENT_FEE,
-    PERFORMANCE_FEE,
-    "Ribbon stETH Theta Vault",
-    "rstETH-THETA",
-    pricer.address,
-    strikeSelection.address,
-    PREMIUM_DISCOUNT,
-    AUCTION_DURATION,
+    {
+      _owner: owner,
```

Comment:
for some of these you prepend with param name, and for some vault deploy scripts like delta you dont

Reply:
Good catch, thanks.